cognee/tests/test_path_config.py

```
@@ -0,0 +1,114 @@
+def test_root_dir_absolute_paths():
+    """Test absolute path handling in root_dir.py"""
+    # Test with absolute path
+    abs_path = "C:/absolute/path" if os.name == 'nt' else "/absolute/path"
+    result = ensure_absolute_path(abs_path, allow_relative=False)
+    assert result == str(Path(abs_path).resolve())
+    
+    # Test with relative path (should fail)
+    rel_path = "relative/path"
+    try:
+        ensure_absolute_path(rel_path, allow_relative=False)
+        assert False, "Should fail with relative path when allow_relative=False"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)